docs/modules/ROOT/nav.adoc

```diff
@@ -282,6 +283,8 @@
 
 .Appendices
 * xref:release-notes/release-notes-history.adoc[History of Release Notes]
+** xref:release-notes/release-notes-23.1.adoc[Payara Enterprise 5.23.1 Release Notes]
+** xref:release-notes/release-notes-23.0.adoc[Payara Enterprise 5.23.0 Release Notes]
```

Comment:
We should remove these references because they are duplicates and Antora creates broken links for them
```suggestion
```